packages/wallet/api/src/lib-wallet.js

```diff
@@ -131,6 +131,22 @@ export function makeWallet({
   const installationMapping = makeMapping('installation');
 
   const brandTable = makeIssuerTable();
+
+  const initBrandToBoardId = async brand => {
+    const { valToBoardId: brandToBoardId } = brandMapping;
```

Comment:
within this function name `initBrandToBoardId`, that the mapping is `brandToBoardId` is implied.

consider 
```suggestion
    const { valToBoardId: map } = brandMapping;
```

Reply:
Removed this method